scripts/zones/Phanauet_Channel/mobs/Stubborn_Dredvodd.lua

```diff
@@ -0,0 +1,24 @@
+-----------------------------------
+-- Area: Phanauet Channel (1)
+--  NM: Stubborn Dredvodd
+-----------------------------------
+mixins = { require("scripts/mixins/job_special") }
+-----------------------------------
+local entity = {}
+
+entity.onMobSpawn = function(mob)
+    mob:setAnimationSub(1) -- Jumping Animation
+    mob:setUntargetable(false)
```

Comment:
What controls this NM spawning when a boat ride is actively happening instead of when its docked at a port?